documentation/operators/src/nodes/network-requester-setup.md

```diff
@@ -263,12 +184,16 @@ It is easy to add new domains and services to your network requester - simply fi
 How to go about this? Have a look in your nym-network-requester config directory:
 
 ```
+# network requester binary
```

Comment:
We have `Network Requester` and `network requester` in this page, as well as `Gateway` and `gateway`. I dont mind which but lets pick one and stick to it for consistency

Reply:
This is our general problem. We shall make a large rule on this and I will implement it all over our books, it will need some time to make it everywhere.